rpmbuild/copr_rpmbuild/automation/fedora_review.py

```diff
@@ -86,7 +86,7 @@ def _filter_results_directory(self):
         # The fedora-review command failed so early that it didn't even create
         # the resultdir. Nothing to do here.
         if not os.path.exists(srcdir):
-            self.log.error("Can't find fedora-review results: %s", srcdir)
+            self.log.error(f"Can't find fedora-review results: {srcdir}")
```

Comment:
The correct way to use logging is to use `%s` so the strings are only evaluated if really used.

Reply:
Reverted this and updated my new log line to match